src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -27,123 +27,132 @@
 
-    private static final double kTimestampToMs = 1000.0 /
-            (1 << kInterArrivalShift) ;
+    //@Todo In webrtc, kTimestampToMs = 1000.0 / (1 << kInterArrivalShift);
+    //Clarify difference.
+    private static final double kTimestampToMs = 1.0 / 90.0;
+
 
     private final Object critSect = new Object();
-    private  TreeMap<Long,Long> ssrcs_ = new TreeMap<Long, Long>();
-    private  ArrayList<Probe> probes_ = new ArrayList<>();
-    private  long totalProbesReceived;
-    private  long firstPacketTimeMs;
-    private  long lastUpdateMs;
+
+    private TreeMap<Long, Long> ssrcs_ = new TreeMap<Long, Long>();
+    private ArrayList<Probe> probes_ = new ArrayList<>();
+    private long totalProbesReceived;
+    private long firstPacketTimeMs;
+    private long lastUpdateMs;
     private RemoteBitrateObserver observer_;
-    private AimdRateControl remoteRate  = new AimdRateControl();
+    private AimdRateControl remoteRate = new AimdRateControl();
     private InterArrival interArrival;
     private OveruseEstimator estimator;
     private OveruseDetector detector;
-    private RateStatistics incomingBitrate ;
+    private RateStatistics incomingBitrate;
     private boolean incomingBitrateInitialized;
     private AbsSendTimeEngine absoluteSendTimeEngine;
 
     public RemoteBitrateEstimatorAbsSendTime(RemoteBitrateObserver observer,
-                                           AbsSendTimeEngine absSendTimeEngine)
+                                             AbsSendTimeEngine absSendTimeEngine)
     {
         super(RTPPacketPredicate.INSTANCE);
         this.absoluteSendTimeEngine = absSendTimeEngine;
         this.observer_ = observer;
         this.interArrival = new InterArrival(90 * kTimestampGroupLengthMs,
-                kTimestampToMs,true);
+            kTimestampToMs, true);
         this.estimator = new OveruseEstimator(new OverUseDetectorOptions());
         this.detector = new OveruseDetector(new OverUseDetectorOptions());
-        this.incomingBitrate = new RateStatistics(kBitrateWindowMs,8000);
+        this.incomingBitrate = new RateStatistics(kBitrateWindowMs, 8000F);
         this.incomingBitrateInitialized = false;
         this.totalProbesReceived = 0;
         this.firstPacketTimeMs = -1;
         this.lastUpdateMs = -1;
-        logger.info("; RemoteBitrateEstimatorAbsSendTime: Instantiating.");
+        if (logger.isDebugEnabled())
+        {
+            logger.debug("RemoteBitrateEstimatorAbsSendTime: Active.");
+        }
     }
 
-    private <K,V> List<K> Keys(TreeMap<K,V> _map)
+    private <K, V> List<K> Keys(TreeMap<K, V> _map)
     {
         ArrayList<K> keys = new ArrayList<K>();
-        for(Map.Entry<K,V> entry : _map.entrySet())
+        for (Map.Entry<K, V> entry : _map.entrySet())
         {
             keys.add(entry.getKey());
         }
         return keys;
     }
 
-    private long ConvertMsTo24Bits(long timeMs)
+    public long ConvertMsTo24Bits(long timeMs)
```

Comment:
does this operate on an rtp timestamp?  we have some helpers in RTPUtils for this sort of thing, could we add this there?